ReadCourseData.py

```diff
@@ -37,18 +69,24 @@ def main():
 			course_raw_data = json.load(f)
 
 	clist = []
+	dlist = []
 	total_course = 0
 	for department in course_raw_data["2018 Winter De Anza"]["CourseData"]:
 		dep_course_num = 0
+		temp_dept = Department(department)
 		for c in course_raw_data["2018 Winter De Anza"]["CourseData"][department]:
 			dep_course_num += 1
-			temp_course = Course(c["CRN"], c["Crse"], c["Sec"], c["Cmp"], c["Cred"],
-							 c["Title"], c["Days"], c["Time"], c["Cap"], c["Act"], 
-							 c["WL Cap"], c["WL Act"], c["Instructor"], c["Date"],
-							 c["Location"], c["Attribute"], c["Lab Time"])
+			temp_course = Course(c["CRN"], c["CRN"], c["Crse"], c["Sec"], c["Cmp"], c["Cred"],
+							 c["Title"], c["Days"], c["Time"][:8], c["Time"][9:], c["Cap"], 
+							 c["WL Cap"], c["Instructor"], c["Date"][:5], c["Date"][6:],
+							 c["Location"], c["Attribute"])
 			clist.append(temp_course)
+			interesting_name = temp_dept.deptName + " " + temp_course.course_num + " " + temp_course.course_title 
```

Comment:
what is `interesting_name` mean?
try to use `"{} {} {}".format(var1,var2,var3)` style